src/schemas/authSchemas.ts

```diff
@@ -4,6 +4,21 @@ import { INVALID_EMAIL_ERROR, MIN_LENGTH_ERROR } from '/src/schemas/const';
 
 import { REQUIRED_ERROR, SPECIAL_CHARACTERS_ERROR } from './validationMessages';
 
+export interface LoginSchemaInterface {
```

Comment:
As I wrote in our `projects` channel, if some types or interfaces are intended to be used in different files, let's create a separate folder for them. I suggested `models` but the name could definitely be different, like `schemas`. Then inside there, there could be a file named e.g. `Auth.d.ts` and it could contain the 3 interfaces that you declared here.